specification.markdown

```diff
@@ -779,7 +779,7 @@ description: The version of the DocTypeExtension. Different DocTypeExtensionVers
 
 ## Global Elements
 
-EBML allows some special Elements to be found within more than one parent in an EBML Document. These Elements are called Global Elements. There are 2 Global Elements that can be found in any EBML Document: the CRC-32 Element and the Void Element. An EBML Schema MAY add other Global Elements to the format it defines.
+EBML allows some special Elements to be found within more than one parent in an EBML Document or optionally at the Root Level of an EBML Document. These Elements are called Global Elements. There are 2 Global Elements that can be found in any EBML Document: the CRC-32 Element and the Void Element. An EBML Schema MAY add other Global Elements to the format it defines.
```

Comment:
I think we need to be more precise with the `EBML Header` and the `EBML Body`. For example `Void` MUST NOT be the first element of the EBML Header. In fact it MUST NOT be found at the root of the `EBML Header`. If it's at the root (ie after the `<EBML/>`) it should be considered as part of the `EBML Body`. And CRC-32 cannot be at the root (I disagree with @JeromeMartinez on that). So this text should be:

> or optionally at the Root Level of an EBML Body

instead of

> or optionally at the Root Level of an EBML Document